.gitignore

```diff
@@ -176,5 +176,9 @@ onnx_models/
 # Generated images
 generated_imgs/
 
+# Custom model related artefacts
+apps/stable_diffusion/src/utils/resources/variants.json
```

Comment:
Do we generate this file ?

Reply:
Yes, Anush.
This is used to serve us two purpose :-
1. Fetch hf_model_id once the session starts - just like we fetch already compiled vmfbs.
2. Fetch hf_model_id to bypass the retry mechanism in case the vmfbs are deleted - saves the user the turnaround time involved in retry method. Therefore even if we use `--clear_all`, we can directly use the knowledge stored in this file and compile the sub-modules directly without any hassle.